widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.kt

```diff
@@ -219,12 +219,7 @@ internal class ChatController(
                 return
             }
 
-            var initChatState = chatState.initChat(companyName, queueId, visitorContextAssetId)
-            if (isSecureEngagement) {
-                initChatState = initChatState.setSecureMessagingState()
-            }
-            prepareChatComponents()
-            emitViewState { initChatState }
```

Comment:
So, there is no need to execute these two lines now, correct?

Reply:
these two lines were moved to the next step (after the chat history loaded) of chat initialization.